charts/rancher-istio/v1.1.1/charts/kiali/values.yaml

```diff
@@ -5,7 +5,7 @@ enabled: false
 replicaCount: 1
 image:
   repository: docker.io/kiali/kiali
-  tag: v0.16
+  tag: v0.17
   proxy:
```

Comment:
We test 0.16 before when we upgrade from 0.14 to 0.16, some UI pages are broken, need a test.